daemon/cmd/endpoint.go

```diff
@@ -573,7 +583,7 @@ func putEndpointIDHandler(d *Daemon, params PutEndpointIDParams) (resp middlewar
 
 	ep.Logger(daemonSubsys).Info("Successful endpoint creation")
 
-	return NewPutEndpointIDCreated()
+	return NewPutEndpointIDCreated().WithPayload(ep.GetModel())
```

Comment:
Heads up for API reviewers that this will increase the size of the endpoint creation response for every Pod creation / CNI ADD event, but I don't think this should be too costly.